src/sagemaker/tensorflow/processing.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2019-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Nitpick: replace `Copyright xxxx-xxxx Amazon.com, Inc. or its affiliates. All Rights Reserved.` with `Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.`. See #2532 and #2531.